packages/flame/lib/src/components/position_component.dart

```diff
@@ -135,20 +206,106 @@ abstract class PositionComponent extends BaseComponent {
   PositionComponent({
     Vector2? position,
     Vector2? size,
-    this.angle = 0.0,
+    double angle = 0.0,
     this.anchor = Anchor.topLeft,
```

Comment:
If the anchor is changed we also need to recalculate the matrix right?
(Couldn't comment on the correct place since nothing was changed around there)

Reply:
Yes, you're right, since the transform depends on the anchor point. I'll add a setter for the anchor.